lib/widgets/screens/new_profile/new_profile_screen.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:mm_flutter_app/data/models/user/user.dart';
+import 'package:mm_flutter_app/data/models/user/user_provider.dart';
+import 'package:mm_flutter_app/widgets/atoms/profile_page_header.dart';
+import 'package:mm_flutter_app/widgets/atoms/profile_text_section.dart';
+import 'package:mm_flutter_app/widgets/molecules/profile_page_social_chips.dart';
+
+String placeholderImage =
+    'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png';
+
+class NewProfileScreen extends StatelessWidget {
+  const NewProfileScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
There should only be a single Scaffold, at the App level. Individual screens should then be displayed in the body of said Scaffold by using the router. Try using a column if you want to arrange widgets vertically.

Reply:
Sounds good. Thanks for the suggestion